lib/view/formulier/InvoerVelden.class.php

```diff
@@ -848,7 +849,7 @@ public function getHtml() {
 
 }
 
-abstract class RequiredEntityField extends EntityField {
+class RequiredEntityField extends EntityField {
```

Comment:
Aangezien PersistentEntity abstract is en je die moet extenden met je eigen class adviseer ik dat hier ook te doen voor consistentie. (Please revert this.)